docs/README.md

```diff
@@ -200,14 +200,14 @@ These are the current limitations in Gopherciser:
   * Variance waterfall chart
   * P&L pivot chart object
   * Trellis container extension
-  * Chart suggestions (that is, auto-charts) for box plot, distribution plot, and histogram objects (to get correct measurements for these types of objects, disable chart suggestions in each such object before starting your test)
+  * Chart suggestions (that is, auto-charts) are supported, however only if the objects where created in Qlik Sense June 2020 or later. For auto-chart objects created in earlier version, this needs to be updated manually in your app   
```

Comment:
Suggestion: 
"Chart suggestions (that is, auto-charts) are supported, but only if the objects were created with Qlik Sense Enterprise June 2020 or later. Auto-chart objects created with earlier versions have to be manually updated in your app."